editor/src/viewport_tools/vector_editor/shape_editor.rs

```diff
@@ -205,12 +203,12 @@ impl ShapeEditor {
 
 	/// Deselect all anchors from the shapes the manipulation handler has created
 	pub fn deselect_all_points(&self, responses: &mut VecDeque<Message>) {
-		responses.push_back(DocumentMessage::DeselectAllVectorPoints.into());
+		responses.push_back(DocumentMessage::DeselectAllManipulatorPoints.into());
 	}
 
 	/// Iterate over the shapes
-	pub fn iter<'a>(&'a self, document: &'a Document) -> impl Iterator<Item = &'a VectorShape> + 'a {
-		self.selected_layers.iter().flat_map(|layer_id| document.layer(layer_id)).filter_map(|shape| shape.as_vector_shape())
+	pub fn iter<'a>(&'a self, document: &'a Document) -> impl Iterator<Item = &'a Subpath> + 'a {
+		self.selected_layers.iter().flat_map(|layer_id| document.layer(layer_id)).filter_map(|shape| shape.as_subpath())
 	}
 
 	/// Find a point that is within the selection threshold and return an index to the shape, anchor, and point
```

Comment:
Several lines below is this comment:
```rs
// Find the closest control point among all elements of shapes_to_modify
```
The usage of the word "control point" is incorrect here.